recipes/ploomber-core/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "ploomber-core" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ploomber-core-{{ version }}.tar.gz
+  sha256: e3edf774900ae3c4f946bd2dea8fd123a741acf5136614233f15df9cde905c0e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - click
+    - posthog
+    - python >=3.6
+    - pyyaml
+
+test:
+  imports:
+    - ploomber_core
+  commands:
+    - pip check
+  requires:
+    - pip
```

Can we run the unit tests?

```suggestion
  commands:
    - pip check
    - pytest tests/
  requires:
    - pip
    - pytest
  source_files:
    - tests/
```

---

we run the tests on the main [repo](https://github.com/ploomber/core/actions), I thought of skipping them here to keep runtime short

---

Hi @conda-forge/staged-recipes, any feedback here? Can we proceed with the existing recipe? We're running the test suite already in our CI system, so we can skip it here and save some build time.

---

Unit tests are not necessary in this case. Just packaging tests are sufficient.